tools/buildmgr/test/testinput/Examples/GCC/Build_GCC/CMakeLists.txt.ref

```diff
@@ -17,7 +17,7 @@ set(MVE NO_MVE)
 set(BYTE_ORDER Little-endian)
 set(AS_FLAGS_GLOBAL "-Wa,--gdwarf-2 -mthumb")
 set(CC_FLAGS_GLOBAL "-O -Wall -gdwarf-2 -mapcs-frame -mthumb")
-set(LD_FLAGS_GLOBAL "--entry=Reset_Handler --specs=nosys.specs -mcpu=cortex-m3 -mthumb")
+set(LD_FLAGS_GLOBAL "--entry=Reset_Handler --specs=nosys.specs -mcpu=cortex-m3 -mthumb -static")
```

Comment:
I have a question. In general, does the order of the flags matter?
@yvesll @brondani

Reply:
Yes it would be important to preserve the flags order. For example single-pass linkers may need the libraries in the correct order to be able to find the symbols.

Reply:
So, we must keep the order like that in csolution/cproject file. If LINK-CPP was added before LINK in , flags under LINK-CPP should also be added before LINK in LD_FLAGS_GLOBAL, is my understanding correct? @brondani

Reply:
@yvesll Keeping the relative order between `Link` and `Link-CPP` as they are in `yml` files can be tricky because the `misc` field is accepted in several places. I would say it would be predictable enough when `Link` flags appear in the command line before the `Link-CPP` flags as it is in the [doc](https://github.com/Open-CMSIS-Pack/devtools/blob/main/tools/projmgr/docs/Manual/YML-Input-Format.md#misc).